src/api/app/views/shared/_involved_users.html.haml

```diff
@@ -0,0 +1,97 @@
+- user_is_maintainer = (@package && User.current.can_modify_package?(@package)) || (@project && User.current.can_modify_project?(@project.api_obj))
+- data = { 'save-user' => url_for(action: 'save_person'),
+-          'remove' => url_for(action: 'remove_role'),
+-          'save-group' => url_for(action: 'save_group'),
+-          'project' => @project.name }
+- data[:package] = @package.name if @package
+= content_tag(:div, id: 'involved_users', data: data) do
+  - if @users.present?
+    %h3
+      User Roles #{image_tag 'ajax-loader.gif', id: 'user_spinner', class: 'hidden'}
+    %table.compact.stripe#user-table
+      %thead
+        %tr
+          %td Username
+          - @roles.each do |role|
+            %td= role.title.capitalize
+          %td
+      %tbody
+        - @users.each do |user|
+          %tr{ id: "user-#{valid_xml_id(user.login)}" }
+            %td= user_with_realname_and_icon(user)
+            - @roles.each do |role|
+              = content_tag(:td, data: { user: user.login, role: role.title, type: 'user' }) do
```

Comment:
This could be like `%td{ data: { user: user.login, role: role.title, type: 'user' }}`